build/zip.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# Copyright 2013 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import argparse
+import zipfile
+import os
+import sys
+
+
+def _zip_dir(path, zip_file, prefix):
+  path = path.rstrip('/\\')
```

Comment:
Not sure why this is necessary.

Reply:
So that I can `root.replace` using it and not end up with extra slashes.

Reply:
(in case someone gives a directory as the argument and includes the extra slash)